include/dynamatic/Support/RTL.h

```diff
@@ -465,7 +465,7 @@ class RTLMatch {
   /// output directory while generated components are produced by the
   /// user-provided generator.
   LogicalResult concretize(const RTLRequest &request, StringRef dynamaticPath,
-                           StringRef outputDir) const;
+                           StringRef outputDir, HDL hdl) const;
```

Comment:
There is no need to pass that as an argument. The `RTLMatch` instance contains a reference to the matched `RTLComponent` which itself references a HDL. From within the method you can do

```cpp
HDL hdl = component.hdl;
```